standard/delegates.md

```diff
@@ -207,6 +207,43 @@ Once instantiated, a delegate instance always refers to the same invocation list
 
 > *Note*: Remember, when two delegates are combined, or one is removed from another, a new delegate results with its own invocation list; the invocation lists of the delegates combined or removed remain unchanged. *end note*
 
+> *Note*: Delegate combination and removal is known to fail with an exception in some implementations when the delegate types involved at runtime are different due to variant conversion ([§18.2.3.3](interfaces.md#18233-variance-conversion). The following example demonstrates a situation which may fail.
```

Comment:
Maybe something along the lines of:

> *Note*: At the time of writing delegate combination...

Rationale: Knowing when an implementation fails to meet the Standard is clearly useful to the (potential) user of that implementation. However whether it is the role of the Standard to do that is less obvious; the implementation’s own documentation should, an Annotated Standard might, but the Standard itself? If the Standard is to do so then indicating the *Note* refers to a point in time reduces the rather definitive tone – after all this anonymous implementation could become compliant (in a ideal world) surely?